### DISCUSSION TITLE
YAML Deny List - Case Sensitivity?

### DISCUSSION THREAD

**Discussion by TalkTomeG00se** [microsoft/presidio#1191] [Q&A] [answered]
I'm adding custom recognizers via a .yaml file and I was wondering if it's possible when implementing the `deny_list` and adding terms, if I can make the terms case insensitive? I anticipate adding numerous terms and don't want to have all permutations of lower case and upper case for each term. Thank you!

**Comment by omri374** [comment, marked as answer]
Hi, we will be working on adding this capability soon. See also https://github.com/microsoft/presidio/issues/1029

**Reply by TalkTomeG00se** [reply]
That's great, I appreciate the info and the ongoing work, thank you!

**Reply by omri374** [reply]
Please check again. A new version has been released
